pandora/tagger.py

```diff
@@ -278,6 +278,18 @@ def setup_to_train(self, train_data=None, dev_data=None, test_data=None,
         if self.include_test:
             self.test_contexts = self.pretrainer.transform(
                 tokens=self.test_tokens)
+        try:
+            del train_transformed
+        except:
+            pass
+        try:
+            del dev_transformed
+        except:
+            pass
+        try:
+            del test_transformed
+        except:
+            pass
```

Comment:
Seems good !
I'd just add a comment for later : it is considered good practice to declare variable before an `if` to ensure that you do not forgot the variable after and do not have to resort too much to try/else. eg : 

```python
         test_transformed = None
         if self.include_test:
               test_transformed = self.preprocessor.transform(
```